pavelib/assets.py

```diff
@@ -636,6 +636,30 @@ def collect_assets(systems, settings):
         print("\t\tFinished collecting {} assets.".format(sys))
 
 
+def execute_compile_sass(args):
+    """
+    Construct django management command compile_sass (defined in theming app) and execute it.
+    Args:
+        args: command line argument passed via update_assets command
+    """
+    for sys in args.system:
+        options = ""
+        options += " --themes " + " ".join(args.themes) if args.themes else ""
+        options += " --themes-dir=" + args.themes_dir if args.themes_dir else ""
+        options += " --debug" if args.debug else ""
+
+        sh(
```

Comment:
What's the reasoning behind calling out to a management command that in turn just calls back to a Paver command? Why do we need a management command in addition to the Paver command? If we need a management command, then why do we also need a Paver command? This seems overly convoluted to me.